File: c/decision/even-odd.c
#include <stdio.h>
int main(void)
{   //function block
    int n;
    printf("Enter a number: ");
    scanf("%d", &n);
    if(n % 2 == 0)
    {   //if block (a group of statements)
        printf("Even Number\n");
    }
    else
    {   //else block
        printf("Odd Number\n");
    }
    return 0;
}

File: java/no-package/decision/EvenOdd.java

import java.util.Scanner;

public class EvenOdd {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.printf("Enter a number: ");
        int n = scanner.nextInt();

        if(n % 2 == 0)
            System.out.printf("Even Number\n");
        else
            System.out.printf("Odd Number\n");
    }
}